book.ru

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Since a book has many rentals and rentals will be created for a book, the getters and setters for rentals should be present.